### PR TITLE
Performance improvement by eliminating `map` before  `any` or `all`

### PR DESCRIPTION
**Pull request by tiemvanderdeure**
Improve the performance of a bunch of functions by not calling `map`, but just using `all(f, args)`

### CODE REVIEW

**Comment by rafaqz** [comment]
`map` before `all` makes tuples faster but iterables slower, so be careful not to remove it from tuples!

File: src/methods/extract.jl [hunk 1]
```diff
@@ -225,7 +225,7 @@ end
         _dimcoord(d, point)
     end
     # If any are coordinates missing, also return missing for everything
-    if any(map(ismissing, coords))
+    if any(ismissing, coords)
```
Comment (rafaqz) [inline src/methods/extract.jl:228]: `coords` is a Tuple too

File: src/methods/extract.jl [hunk 2]
```diff
@@ -258,7 +258,7 @@ end
...
```
Comment (rafaqz) [inline src/methods/extract.jl:261]: And here

**Comment by tiemvanderdeure** [comment]
so it really was just the one in aggregate :)

**Pull request merged**